builtin/providers/aws/resource_aws_api_gateway_method_test.go

```diff
@@ -154,6 +187,108 @@ func testAccCheckAWSAPIGatewayMethodDestroy(s *terraform.State) error {
 	return nil
 }
 
+const testAccAWSAPIGatewayMethodConfigWithCustomAuthorizer = `
+resource "aws_api_gateway_rest_api" "test" {
+  name = "test"
```

Comment:
Probably a nitpick, but it may be better to give the API unique name, e.g. `tf-acc-test-authorizer`. Should these tests leak (i.e. leave undeleted resources behind) we will be able to track the test more easily.